tracer/src/Datadog.Trace/Generated/netcoreapp3.1/Datadog.Trace.SourceGenerators/Datadog.Trace.SourceGenerators.TagsListGenerator.TagListGenerator/GraphQLTagsBase.g.cs

```diff
@@ -0,0 +1,124 @@
+﻿// <auto-generated/>
```

Comment:
Now that you have removed the GraphQLTagsBase and HotChocolateTags you can remove all genereated files as well.